internal/cmd/gocdk/biome_add.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	slashpath "path"
+	"path/filepath"
+	"text/template"
+
+	"golang.org/x/xerrors"
+)
+
+func biomeAdd(ctx context.Context, pctx *processContext, args []string) error {
+	// TODO(clausti) interpolate launcher from one supplied as a flag
+	f := newFlagSet(pctx, "demo")
+	usageMsg := "gocdk biome add BIOME_NAME"
+	if err := f.Parse(args); xerrors.Is(err, flag.ErrHelp) {
+		return nil
+	} else if err != nil {
+		return usagef("%s: %w", usageMsg, err)
+	}
+	if f.Args()[0] != "add" {
+		return usagef("%s: expected add, got %v", usageMsg, f.Args()[0])
+	}
+	newName := f.Args()[1]
```

Comment:
nit: FlagSet has a method Arg(i int) string which lets you get the argument without going through the slice access.

```suggestion
	newName := f.Arg(1)
```